cacerts/jksutil/certificate.go

```diff
@@ -0,0 +1,28 @@
+package jksutil
```

Comment:
Please add the license header found [here](https://github.com/GoogleContainerTools/distroless/blob/master/base/testdata/check_certs.go) (with year 2020).